src/bot/utils/load_env.py

```diff
@@ -0,0 +1,7 @@
+from dotenv import load_dotenv
+import os
+
+def load_env_dev() -> None:
```

Comment:
Shared utilities would be a good thing rather than duplicating code.

This might be impossible to do cleanly but it would be nice.

Can you have a think of how this could be done and let me know?

Reply:
It actually depends on the deployment strategy of the api and the bot.

Reply:
I will discuss this with you via voice